spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 end
 Capybara.javascript_driver = :poltergeist
 
-Capybara.default_max_wait_time = 10
+Capybara.default_max_wait_time = 30
```

Comment:
I'm still getting the occasional timeouts and Poltergeist problems locally, but it's not as bad as under CI. I added a ternary operator for now and will keep an eye on tests failing locally.